app/decorators/school_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class SchoolDecorator < Draper::Decorator
+  delegate_all
+
+  def formatted_address
+    address_parts = [address1, address2, address3, town, postcode]
+    address_parts.reject!(&:blank?)
+    address_parts.join("<br/>").html_safe
```

Comment:
![AKOGxCJ0ksM7c5zzIxWg0kDOL-llpMlKpZbcvjohFHs ](https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/1c2e58a9-73de-46db-8df7-db15b5604381)

I don't really have a strong opinion on this, not a blocker just reminded me of css memes 😄

Reply:
Seemed like the best way to conditionally format the address from the given attributes.
Given then some of them may be `nil`, specifying each attribute on the page would leave large gaps.
Happy if you have another suggestion.